package.json

```diff
@@ -1,95 +1,97 @@
 {
-  "name": "cardsforhumanity",
-  "description": "A Game For Horrible People Desperately Trying To Do Good",
-  "version": "1.0.0",
-  "private": false,
-  "author": "cardsforhumanity",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
-  },
-  "engines": {
-    "node": "8.9.0",
-    "yarn": "1.3.2"
-  },
-  "scripts": {
-    "postinstall": "node node_modules/gulp/bin/gulp install",
-    "heroku-postbuild": "node node_modules/gulp/bin/gulp build",
-    "start": "yarn heroku-postbuild && node node_modules/gulp/bin/gulp",
-    "test": "NODE_ENV=test && node node_modules/gulp/bin/gulp test",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "coveralls": "nyc --reporter=lcov --reporter=text-lcov npm test",
-    "test:run": "karma start ./dist/karma.config.js"
-  },
-  "dependencies": {
-    "async": "^2.6.0",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "bcryptjs": "^2.4.3",
-    "body-parser": "^1.18.2",
-    "bower": "^1.8.2",
-    "browser-sync": "^2.18.13",
-    "compression": "^1.7.1",
-    "connect": "^3.6.5",
-    "connect-flash": "^0.1.1",
-    "connect-mongo": "1.3.0",
-    "connect-timeout": "^1.9.0",
-    "console-stamp": "^0.2.6",
-    "cookie-parser": "^1.4.3",
-    "cookie-session": "^2.0.0-beta.3",
-    "coveralls": "^3.0.0",
-    "csurf": "^1.9.0",
-    "dotenv": "^4.0.0",
-    "errorhandler": "^1.5.0",
-    "express": "^4.16.2",
-    "express-session": "^1.15.6",
-    "forever": "^0.15.3",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^7.0.0",
-    "gulp-bower": "^0.0.13",
-    "gulp-cli": "^1.4.0",
-    "gulp-eslint": "^4.0.0",
-    "gulp-exit": "^0.0.2",
-    "gulp-mocha": "^4.3.1",
-    "gulp-nodemon": "^2.2.1",
-    "gulp-sass": "^3.1.0",
-    "jade": "^1.11.0",
-    "jsonwebtoken": "^8.1.0",
-    "mean-logger": "^0.0.1",
-    "method-override": "^2.3.10",
-    "mongodb": "^2.2.33",
-    "mongodb-migrate": "^2.0.2",
-    "mongoose": "^4.13.5",
-    "morgan": "^1.9.0",
-    "nyc": "^11.3.0",
-    "passport": "^0.4.0",
-    "passport-facebook": "^2.1.1",
-    "passport-github": "^1.1.0",
-    "passport-google-oauth": "^1.0.0",
-    "passport-local": "^1.0.0",
-    "passport-twitter": "^1.0.4",
-    "response-time": "^2.3.2",
-    "serve-favicon": "^2.4.5",
-    "serve-index": "^1.9.1",
-    "serve-static": "^1.13.1",
-    "session": "^0.1.0",
-    "sinon": "^4.1.2",
-    "socket.io": "^2.0.4",
-    "socket.io-client": "^2.0.4",
-    "underscore": "^1.8.3",
-    "vhost": "^3.0.2",
-    "view-helpers": "^0.1.5"
-  },
-  "devDependencies": {
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.8.0",
-    "jasmine": "^2.8.0",
-    "jasmine-core": "^2.8.0",
-    "karma": "^1.7.1",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-jasmine": "^1.1.0",
-    "should": "^13.1.3",
-    "supertest": "^3.0.0"
-  }
+    "name": "cardsforhumanity",
```

Comment:
The package.json should be using 2 spaces for a tab instead of 4 spaces. The spacing of the deletions in the file was ok.

Always, choose `incoming changes or both` when fixing conflicts and ensure everything works well together. The deletions have `babel`, `babel-cli`, `gulp-babel` etc that was added to the package.json from staging but missing in this branch.

Revert to staging package.json and then add the new package you added  in this branch